cmd/operator-sdk/generate/bundle/bundle.go

```diff
@@ -28,6 +28,76 @@ import (
 	"github.com/operator-framework/operator-sdk/internal/generate/collector"
 )
 
+const (
+	//nolint:lll
+	examples = `
+  # Using the example 'memcached-operator' and assuming a directory structure
+  # similar to the following exists:
+  $ tree api/ config/
+  api/
+  └── v1alpha1
+      ├── groupversion_info.go
+      ├── memcached_types.go
+      └── zz_generated.deepcopy.go
+  config/
+  ├── bundle
+  │   └── kustomization.yaml
+  ├── crd
+  │   ├── bases
+  │   │   └── cache.my.domain_memcacheds.yaml
+  │   ├── kustomization.yaml
+  │   ├── kustomizeconfig.yaml
+  │   ...
+  ├── default
+  │   ├── kustomization.yaml
+  │   ...
+  ├── manager
+  │   ├── kustomization.yaml
+  │   └── manager.yaml
+  ...
+
+  # Generate bundle files and build your bundle image with this 'make' recipe:
+  $ export USERNAME=<your registry username>
+  $ make bundle-build BUNDLE_IMG=quay.io/$USERNAME/memcached-operator-bundle:v0.0.1
+
+  # The above recipe runs the following commands manually. First it creates bundle
+  # manifests, metadata, and a bundle.Dockerfile:
+  $ make manifests
+  /home/user/go/bin/controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+  operator-sdk generate bundle -q --kustomize
+
+  Display name for the operator (required):
+  > memcached-operator
```

Comment:
Is required to add its output?

Reply:
Nope, just helps the user know what expected output should be. I can see this drifting from actual output though.